cmake/FindUring.cmake

```diff
@@ -0,0 +1,24 @@
+# - Find uring
+#
+# URING_INCLUDE_DIR - Where to find liburing.h
+# URING_LIBRARIES - List of libraries when using uring
+# uring_FOUND - True if uring found
+
+find_path(URING_INCLUDE_DIR liburing.h)
+find_library(URING_LIBRARIES liburing.a liburing)
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(uring DEFAULT_MSG URING_LIBRARIES URING_INCLUDE_DIR)
+
+if(uring_FOUND AND NOT TARGET uring::uring)
+  add_library(uring::uring UNKNOWN IMPORTED)
+  set_target_properties(uring::uring PROPERTIES
+    INTERFACE_INCLUDE_DIRECTORIES "${URING_INCLUDE_DIR}"
+    IMPORTED_LINK_INTERFACE_LANGUAGES "C"
+    IMPORTED_LOCATION "${URING_LIBRARIES}")
+endif()
+
+message("URING_INCLUDE_DIR: " ${URING_INCLUDE_DIR})
+message("URING_LIBRARIES: " ${URING_LIBRARIES})
+
+mark_as_advanced(URING_INCLUDE_DIR URING_LIBRARIES)
```

Comment:
I don't understand what is it doing.
In some examples it before `if (...` in other after

Reply:
I copied it from another `Find<...>.cmake` :-)